go/src/koding/klient/machine/mount/index/index_std.go

```diff
@@ -0,0 +1,14 @@
+// +build !darwin,!linux
```

Comment:
This file stubs `ctimeFromSys` for platforms other than darwin/linux (e.g. Windows). Just to make it possible to build the index package.